configure.ac

```diff
@@ -239,6 +239,7 @@ OPTION_DEFAULT_DISABLE([amqp], [ENABLE_AMQP])
 
 dnl Options for contributed stores
 OPTION_DEFAULT_DISABLE([tutorial-store],[ENABLE_TUTORIAL_STORE])
+OPTION_DEFAULT_DISABLE([timescale-store],[ENABLE_TIMESCALE_STORE])
```

Comment:
I would suggest not doing this.

Additionally, this plugin needs autoconf tests for libpq, libecpg, and the header libpq-fe.h.  See my comments about the ldms/src/contrib/store/timescale/Makefile.am for a suggestion on how to test for the two libraries.